doc/content/modules/user-manual/pages/release-notes/2025.6.0.adoc

```diff
@@ -186,6 +186,10 @@ image::release-notes-GV-actionDefinition-parameters.png[Parameters compartment o
 
 image::gv-parts-perform-actions-compartment.png[perform actions compartment in parts, width=70%, align="center"]
 
+- New compartment in `ActionUsage` and `ActionDefinition` to display `PerformActionUsage` in the General view diagram.
```

Comment:
Add a `perform actions` compartment on `ActionUsage` and `ActionDefinition` to display `PerformActionUsage` in the _General View_ diagram.